style.css

```diff
@@ -1,564 +1,563 @@
 * {
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
-  }
-  
-  html {
-    font-size: 62.5%;
-  }
-  
-  body {
-    font-family: 'Lato', sans-serif;
-    overflow-x: hidden;
-  }
-  
-  :root {
-    --main-color: #ec5242;
-  }
-  
-  /* ***************** */
-  
-  /* **** NAVBAR ***** */
-  
-  /* ***************** */
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  overflow-x: hidden;
+}
+
+:root {
+  --main-color: #ec5242;
+}
+
+/* ***************** */
+
+/* **** NAVBAR ***** */
+
+/* ***************** */
+.header-nav {
+  display: none;
+}
+
+.icon {
+  display: none;
+  height: 6.4rem;
+}
+
+.container {
+  padding: 2.5rem;
+}
+
+.navbar {
+  background-color: #e9ecef;
+}
+
+.mobile-menu-icon {
+  width: 2rem;
+}
```

Comment:
Clicking the mobile menu in the about page does not work as expected. Please investigate this🪲

Reply:
@Shisui6 noted